### DISCUSSION TITLE
Webhook inter

### DISCUSSION THREAD

**Discussion by valberaguiar** [HashLoad/horse#378] [Q&A]
Alguém já utilizou o horse para fazer um webhook para o banco inter? Estou com duvida de como informo o certificado disponibilizado por eles (ca.crt).

**Comment by viniciussanchez** [comment]
Bom dia... no Horse em alguns tipos de aplicações, como por exemplo console, você vai ter acesso a propriedade IOHandleSSL, acessando THorse.IOHandleSSL... através dela, você configura seu certificado. Tem um vídeo no Youtube do Igor Bastos, onde ele mostra isso na prática.  Agora se o tipo de aplicação que você estiver utilizando não tiver a propriedade IOHandleSSL, como por exemplo o ISAPI, então, você deve configurar seu certificado digital em algum servidor de aplicação, como o IIS no caso.

**Reply by valberaguiar** [reply]
Minha aplicação é console, mas, não tenho acesso a propriedade IOHandleSSL.

**Reply by viniciussanchez** [reply]
deve estar usando versão antiga do Horse entao

**Reply by valberaguiar** [reply]
Estou usando a versão: HORSE_VERSION = '3.1.4';

**Reply by viniciussanchez** [reply]
Estranho... Delphi e console, usando essa versão 3, você deveria ter acesso a THorse.IOHandleSSL....

**Reply by valberaguiar** [reply]
Esqueci de mencionar meu ambiente, estou sando linux (mint) e lazarus 3.0.

**Reply by viniciussanchez** [reply]
Assim, realmente o modo "console" do Horse para Lazarus, ainda não foi implementado a classe IOHandleSSL. Se tiver interesse poderia estar contribuindo com o projeto, implementando e nos enviando um pull request. Outra possibilidade como falei, seria usar algum servidor de aplicação na frente dele, como um Apache, nginx por exemplo...

**Reply by valberaguiar** [reply]
Muito obrigado pelas informações, vou olhar o código em delphi e ver se consigo replicar em lazarus.
